packages/common/src/auditlogs/auditlogs.js

```diff
@@ -226,15 +226,22 @@ export const AuditLogs = () => {
           onChangeRowsPerPage={newPerPage => onChangePagination(1, newPerPage)}
           onChangeSorting={onChangeSorting}
           selectionState={selectionState}
-          onIssueSelection={onIssueSelection}
           userCapabilities={userCapabilities}
           auditLogColumns={[
             { title: 'Performed by', sortable: false, render: UserDescriptor },
             { title: 'Action', sortable: false, render: ActionDescriptor },
             { title: 'Type', sortable: false, render: TypeDescriptor },
             { title: 'Changed', sortable: false, render: ChangeDescriptor },
             { title: 'More details', sortable: false, render: ChangeDetailsDescriptor },
-            { title: 'Time', sortable: true, render: TimeWrapper }
+            { title: 'Time', sortable: true, render: TimeWrapper },
+            {
+              title: '',
+              sortable: false,
+              render: (item, index) => {
+                const allowsExpansion = !!item.change || item.action.includes('terminal') || item.action.includes('portforward');
+                return ViewDetails(item, index, onIssueSelection, allowsExpansion);
+              }
```

Comment:
this would ideally go into a separate Component or maybe only 
```suggestion
  return <ViewDetails item={item} index={index} onIssueSelection={onIssueSelection} allowsExpansion={allowsExpansion} />;
}
```